inst/tools/r-stripper/stripper

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2016 Daniel C. Dillon
+#
+# r-stripper is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# r-stripper is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with r-stripper.  If not, see <http://www.gnu.org/licenses/>.
+
+if [ -z $R_HOME ]; then
+    R_HOME=`R RHOME`
+fi
+
+if [ "$R_HOME" = "" ]; then
+    echo "Could not find R_HOME."
+    exit 0
+fi
+
+$R_HOME/bin/Rscript inst/tools/r-stripper/stripper.R $1
```

Comment:
Usually zero.  It's probably not necessary to check for it or anything, but better practice.